src/components/PatientRecord/PatientRecord.tsx

```diff
@@ -143,4 +150,39 @@ const Visualizer: FC<VisualizerProps> = ({ resourceType, resourcesByType }) => {
   else return <div>Unsupported Resource</div>;
 };
 
+const PathwayVisualizer: FC = () => {
+  const resources = usePatientRecords().patientRecords;
+  const evaluatedPathway = usePathwayContext().evaluatedPathway;
+  const [criteria, setCriteria] = useState<CriteriaResult | null>(null);
+
+  useEffect(() => {
+    // Create a fake Bundle for the CQL engine and check if patientPath needs to be evaluated
+    const patient = {
+      resourceType: 'Bundle',
+      type: 'searchset',
+      entry: resources.map((r: fhir.Resource) => ({ resource: r }))
+    };
+
+    // Evaluate pathway criteria
+    if (evaluatedPathway) {
+      evaluatePathwayCriteria(patient, evaluatedPathway.pathway).then(criteriaResult =>
```

Comment:
We might want to add a ticket for creating a `CriteriaContext` to avoid recalculating the criteria multiple times in the future.  `PathwayList` also calculates it currently.

Reply:
Yeah I think that fits as part of -203 in the backlog, I'll bump that up